src/app/modules/core/components/footer/footer.component.ts

```diff
@@ -108,47 +128,69 @@ export class FooterComponent extends deepCopy implements OnInit {
+            if(newreach.parameters[7]) {  newreach.parameters[7].value = (reaches[i].parameters[7].value * 0.0000022046).toUSGSvalue(); }            // spill mass from milligrams to pounds
+  
+            newreach.parameters[0].unit.unit = this.units.imperial['discharge']                    // mean annual discharge
+            newreach.parameters[0].unit.abbr = this.abbrev.imperial['discharge']
+            newreach.parameters[1].unit.unit = this.units.imperial['discharge']                    // real-time discharge
+            newreach.parameters[1].unit.abbr = this.abbrev.imperial['discharge']
+            newreach.parameters[2].unit.unit = this.units.imperial['slope']                        // slope
+            newreach.parameters[2].unit.abbr = this.abbrev.imperial['slope']
+            newreach.parameters[3].unit.unit = this.units.imperial['drainageArea']                 // drainage area
+            newreach.parameters[3].unit.abbr = this.abbrev.imperial['drainageArea']
+            newreach.parameters[4].unit.unit = this.units.imperial['distance']                     // reach length
+            newreach.parameters[4].unit.abbr = this.abbrev.imperial['distance']
+            if(newreach.parameters[6]) { newreach.parameters[6].unit.unit = this.units.imperial['distance'] }   // cumulative length
+            if(newreach.parameters[6]) { newreach.parameters[6].unit.abbr = this.abbrev.imperial['distance'] }
+            if(newreach.parameters[7]) { newreach.parameters[7].unit.unit = this.units.imperial['concentration'] }               // spill mass
+            if(newreach.parameters[7]) { newreach.parameters[7].unit.abbr = this.abbrev.imperial['concentration'] }
+  
+            newreach.result.equations.vmax.value = (reaches[i].result.equations.vmax.value * 3.2808399).toUSGSvalue(); // m/s to ft/s
+            newreach.result.equations.v.value = (reaches[i].result.equations.v.value * 3.2808399).toUSGSvalue(); // m/s to ft/s
+  
+            newreach.result.equations.vmax.units = this.abbrev.imperial['velocity']
+            newreach.result.equations.v.units = this.abbrev.imperial['velocity']
+  
+            tempreaches.push(newreach);
+        }
+        this.StudyService.selectedStudy.SpillMass = (this.StudyService.selectedStudy.SpillMass * 0.453592).toUSGSvalue();
+        this.reaches = tempreaches;
+      } else {
+        this.reaches = reaches;
+      } // keep existing metric unit
+
+    } else { // planning methodd
       this.reaches = reaches;
-    } // keep existing metric units
+    }
+  }
+
+  private wm(msg: string, title?: string, timeout?: number) {
+    try {
+      let options: Partial<IndividualConfig> = null;
+      if (timeout) { options = { timeOut: timeout }; }
+      if (timeout == 0) {
+        options = {
+          disableTimeOut : true,
+          timeOut: 0,
+          extendedTimeOut: 0,
+          tapToDismiss: false
```

Comment:
I did set them to allow the user to dismiss and set up a "Start over" button as well. Pete wanted the "ToT not computed for rain-drop trace" message to be pretty sticky but we'll see what he says on our next call. Also, I made the error messages sticky because the user CANNOT continue if the initial NLDI trace doesn't work - another reason for adding the "Start Over" button.